packages/sdk/src/__tests__/handlePanic.test.ts

```diff
@@ -0,0 +1,135 @@
+import mkdir from 'make-dir'
+import { stdin } from 'mock-stdin'
+import { join, resolve } from 'path'
+import prompt from 'prompts'
+import stripAnsi from 'strip-ansi'
+import tempy from 'tempy'
+
+import { ErrorArea, jestConsoleContext, jestContext, RustPanic } from '..'
+import * as sendPanicUtils from '../sendPanic'
+import * as githubUtils from '../utils/getGithubIssueUrl'
+import { handlePanic } from '../utils/handlePanic'
+
+const keys = {
+  up: '\x1B\x5B\x41',
+  down: '\x1B\x5B\x42',
+  enter: '\x0D',
+  space: '\x20',
+}
+const sendKeystrokes = async (io) => {
+  io.send(keys.down)
+  io.send(keys.enter)
+  await delay(10)
+}
+// helper function for timing
+const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms)) // Mock stdin so we can send messages to the CLI
+
+const testRootDir = tempy.directory()
+
+// eslint-disable-next-line @typescript-eslint/unbound-method
+const oldProcessCwd = process.cwd
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+describe('handlePanic', () => {
+  // testing with env https://stackoverflow.com/a/48042799/1345244
+  const OLD_ENV = process.env
+
+  beforeEach(async () => {
+    jest.resetModules() // most important - it clears the cache
+    process.env = { ...OLD_ENV } // make a copy
```

Comment:
I know this comment is only copied from SO, but: Does this really _make_ a copy? Does that not use the "backup" and overwrite the current env values?

Reply:
Yes, `process.env = { ...OLD_ENV }` actually makes a shallow copy of the environment variables that were set before the test suite actually ran.  `process.env = OLD_ENV` then restores the original environment variables after each test suite has executed.